pkg/helpers/results_chaining_keys.go

```diff
@@ -0,0 +1,5 @@
+package helpers
+
+const (
+	ReadingFieldKey = "FieldKey"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported const ReadingFieldKey should have comment (or a comment on this block) or be unexported](https://app.codacy.com/app/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4570750)